Purchases/Misc/Obsoletions.swift

```diff
@@ -35,7 +35,8 @@ public extension Purchases {
     @available(tvOS, obsoleted: 1, renamed: "restorePurchases(completion:)")
     @available(watchOS, obsoleted: 1, renamed: "restorePurchases(completion:)")
     @available(macOS, obsoleted: 1, renamed: "restorePurchases(completion:)")
-    @objc func restoreTransactions(completion: ((CustomerInfo?, Error?) -> Void)? = nil) {
+    @objc(restoreTransactionsWithCompletionBlock:)
+    func restoreTransactions(completion: ((CustomerInfo?, Error?) -> Void)? = nil) {
```

Comment:
without the `block` bit, these wouldn't get picked up correctly

Reply:
Whoa